documentation/asciidoc/stories/assembly_upgrading_clusters.adoc

```diff
@@ -5,6 +5,7 @@ ifdef::context[:parent-context: {context}]
 
 [role="_abstract"]
 {ispn_operator} handles {brandname} cluster upgrades when new versions become available.
```

Comment:
Cluster upgrades are no longer automatic. Once the user upgrades to an operator that has a new server version available, they must manually set the `spec.version` field to the server version they require. Once this has been updated, the cluster upgrade will proceed.

Reply:
@ryanemerson and do users get notified when new Infinispan versions become available?

Reply:
> @ryanemerson and do users get notified when new Infinispan versions become available?

I think we just have to document this in the release notes. Similarly, it's important that we include a supported Operand table in the docs similar to what strimzi do at https://strimzi.io/downloads/

> Wouldn't it be possible to keep them automatic if the user wouldn't set operand version?

@Crumby Yes it's possible but I'm not sure how desirable automatic cluster upgrades are in practice, especially when the default GracefulShutdown upgrades are configured. Automatic upgrades with GracefulShutdown results in cluster downtime at the point of Operator upgrade (assuming a new Operand version is available), which we have seen is not desired by users and is why the Manual upgrade strategy has been desired by users so far. 

Automatic upgrading is something that we can easily add in the future if there's user demand.

N.B. Now that Operator -> Operand upgrading is decoupled, I would hope that more users will trust the Automatic OLM upgrading of the Operator.

Reply:
Thanks for the info Ryan. I'll add the table for the upstream docs and will follow up with downstream later.